snap/quota/resources.go

```diff
@@ -63,8 +81,24 @@ type Resources struct {
 	CPU     *ResourceCPU     `json:"cpu,omitempty"`
 	CPUSet  *ResourceCPUSet  `json:"cpu-set,omitempty"`
 	Threads *ResourceThreads `json:"thread,omitempty"`
+	Journal *ResourceJournal `json:"journal,omitempty"`
 }
 
+const (
+	// make sure the memory limit is at least 640K, that is the minimum size
+	// we will require for a quota group. Newer systemd versions require up to
+	// 12kB per slice, so we need to ensure 'plenty' of space for this, and also
+	// 640kB seems sensible as a minimum.
+	MemoryLimitMin = 640 * quantity.SizeKiB
+
+	// Systemd specifies that the maximum journal log size is 4GB. There is no
+	// minimum value according to the documentation, but the source code reveals
+	// that the minimum value will be adjusted if it is set lower than the current
+	// usage, but we have selected 64kB to protect against rediciously small values.
+	JournalLimitMin = 64 * quantity.SizeKiB
+	JournalLimitMax = 4 * quantity.SizeGiB
```

Comment:
Thanks for this! And sorry I didn't mention this originally, but AFAICT they don't have to be exported?